CHANGELOG.md

```diff
@@ -4,6 +4,24 @@
 <!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
 <!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
 
+## 0.19.0 (2023-10-02)
+
+### New features
+
+- A `transfer_when_all` sender adaptor has been introduced. ([#792](https://github.com/pika-org/pika/pull/792))
+- We are now testing pika compiled with C++23. ([#779](https://github.com/pika-org/pika/pull/779))
+
+### Breaking changes
+
+- The `PIKA_WITH_DISABLED_SIGNAL_EXCEPTION_HANDLER` CMake option has been removed. This option can be controlled at runtime. ([#773](https://github.com/pika-org/pika/pull/773))
+- The `PIKA_WITH_THREAD_GUARD_PAGE` CMake option has been removed. This option can be controlled at runtime. ([#782](https://github.com/pika-org/pika/pull/782))
```

Comment:
Maybe also here:
```suggestion
- The `PIKA_WITH_THREAD_GUARD_PAGE` CMake option has been removed. This option can be controlled at runtime as before. ([#782](https://github.com/pika-org/pika/pull/782))
```
?

Simply to make it clear that the runtime option was always there.